### PR TITLE
Doesn't work

### PR DESCRIPTION
**Pull request by seankhliao**
standard response to "it doesn't work".

maybe need a nicer way to say this?

### CODE REVIEW

File: cmd/consumer/responses.go [hunk 1]
```diff
@@ -239,6 +239,10 @@ func injectMessageResponses(ma *handler.MessageActions) {
 		`- <https://golang.org/conduct>`,
 		`If you have any questions or concerns please reach out in <#C4U9J9QBT> or email support@gobridge.org.`,
 	)
+
+	ma.HandleStatic("doesn't work", "x doesn't work", []string{"doesnt work", "doesntwork"},
+		`What do you mean it doesn't work? If there's an error, copy it in full. If it's long, use the playground.`,
```
Comment (theckman) [inline cmd/consumer/responses.go:244]: How would you feel about:

```
`Are you able to share more context about what's not working? What did you expect to see, and what did you see instead? If there's an error, are you able to provide it in full and share how you generated that error?`,
`If the error is long, please upload using the Playground: <https://play.golang.org/>`
```
That first sentence feels too wordy, and is maybe better without that middle part being included...

> Are you able to share more context about what's not working? If there's an error, are you able to provide it in full and share how you generated that error?

What are your thoughts?

Comment (seankhliao) [reply cmd/consumer/responses.go:244]: What about
> Can you share more context on what you expected and what you saw instead?
> If there's an error, are you able to provide it in full and share how you generated that error?

Comment (theckman) [reply cmd/consumer/responses.go:244]: I like it. 👍
